src/app/app-routing.ts

```diff
@@ -1,31 +1,29 @@
-import {ActivatedRouteSnapshot, CanActivateFn, ResolveFn, Router, Routes} from '@angular/router';
+import {ActivatedRouteSnapshot, CanActivateFn, Router, Routes} from '@angular/router';
 
 import {CampaignListResolver} from './campaign-list.resolver';
 import {CampaignResolver} from './campaign.resolver';
 import {CharityCampaignsResolver} from './charity-campaigns.resolver';
-import {campaignStatsResolver} from "./campaign-stats-resolver";
-import {highlightCardsResolver} from "./highlight-cards-resolver";
+import {CampaignStatsResolver} from "./campaign-stats-resolver";
```

Comment:
Resolvers became classes and one-per-class mostly because I was not sure about the causes of `inject()` errors. And I'm still not certain if it was necessary but I do find the routes more manageable with this setup so I think it is probably worth keeping regardless.

Reply:
Thanks, commented on this in my main review comment.